app/models/release_service.rb

```diff
@@ -21,8 +22,14 @@ def can_release?
 
   private
 
-  def push_tag_to_git_repository(release)
-    GITHUB.create_release(@project.repository_path, release.version, target_commitish: release.commit)
+  def push_tag_to_git_repository(version, commit)
+    GITHUB.create_release(@project.repository_path, version, target_commitish: commit)
+  end
+
+  def ensure_tag_in_git_repository(tag)
+    Samson::Retry.with_retries [Octokit::NotFound], 3, sleep: 1 do
```

Comment:
is 3 enough? or is 1 second enough? Maybe make ENV configurable for easier adjustment?

Reply:
I'd bet that's enough ... we currently do the new deploy immediately so the slightest wait would fix it